__tests__/main.test.ts

```diff
@@ -1,29 +1,6 @@
-import {wait} from '../src/wait'
-import * as process from 'process'
-import * as cp from 'child_process'
-import * as path from 'path'
 import {expect, test} from '@jest/globals'
 
-test('throws invalid number', async () => {
+test('dummy', async () => {
```

Comment:
Tests would be nice!

Reply:
Added a happy path test in https://github.com/unhack/keygen-action/pull/11/commits/ff9992ae9409113f3d7c8a0d5b90e98ad606184b